packages/styles/scss/patterns/logo-grid/_logo-grid.scss

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+@mixin logo-grid {
+  .#{$prefix}--logo-grid {
+    &__container {
+      @include carbon--make-container;
+
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--layout-06;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__col {
+      @include carbon--make-col-ready;
+    }
+    &__title {
+      margin: auto;
+      width: 45%;
```

Comment:
@mkothur This explicit width seems oddly specific. This is in the designs @wonilsuhibm ?

Reply:
Can I have the design specs for this pattern, Actually I took the reference from the ticket description.

Reply:
I don't think we have design specs for this yet. I'm at the moment thinking based on where the design is that the title here would use the copy from the content section without its title. So close to what we currently have in the `ListSection` pattern.

![image](https://user-images.githubusercontent.com/3793636/71687429-6c49a780-2d63-11ea-9fd4-f970be0fe507.png)